tests/unit/recommenders/datasets/mock/test_movielens.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
Traditionally we have tried to not add too many folder levels, just to reduce the complexity of the repo. Do we really need the mock folder, or can we directly add the test inside datasets?

Reply:
File merged into movielens.